desktop/apps/artwork/components/auction/test/view.coffee

```diff
@@ -136,6 +137,21 @@ describe 'auction', ->
           view.$('.js-artwork-auction-bid').attr('action')
             .should.equal "/artwork/#{@data.artwork.id}"
 
+        it 'renders a disabled "Registration Closed" button', ->
+          @data.accounting = accounting
+          @data.me = {
+            bidders: [{
+              qualified_for_bidding: false,
+              registration_ends_at: moment().subtract(2, 'days').format()
+            }]
+          }
+          view = new @ArtworkAuctionView data: @data
+          view.render()
+          view.$('.artwork-auction__bid-form__button').text()
+            .should.containEql 'Registration Pending'
```

Comment:
Nevermind- maybe just change the `it` above to say "'registration disabled' button"